cfgov/unprocessed/js/organisms/FilterableListControls.js

```diff
@@ -18,157 +17,254 @@ var Notification = require( '../molecules/Notification' );
+  function init( ) {
     _notification = new Notification( _dom );
     _notification.init();
+
+    _initEvents();
+  }
+
+  /**
+   * Initialize FilterableListControls events.
+   */
+  function _initEvents() {
     _form.addEventListener( 'submit', _formSubmitted );
+  }
 
-    var inputs = _validatorDefaults.inputs;
-    var fields;
-    var field;
-    var type;
-    for ( var i = 0, len = inputs.length; i < len; i++ ) {
-      fields = _form.querySelectorAll( inputs[i] );
-      for ( var f = 0, flen = fields.length; f < flen; f++ ) {
-        field = fields[f];
-        type = field.getAttribute( 'type' );
-        if ( type !== 'hidden' &&
-             type !== 'button' &&
-             type !== 'submit' &&
-             type !== 'reset' &&
-             typeof field.getAttribute( 'disabled' ) !== 'undefined' ) {
-          _fields.push( field );
-        }
-      }
-    }
+  /**
+   * Remove Event listeners.
+   */
+  function destroy() {
+    _form.removeEventListener( 'submit', _formSubmitted );
   }
 
   /**
    * Show error notification.
    * @param {Object} event Form submitted event.
-  */
+   */
   function _formSubmitted( event ) {
-    event.preventDefault();
-    var validatedFields = _validateFields( _fields );
+    var validatedFields = _validateFields( [].slice.call( _form.elements ) );
 
     if ( validatedFields.invalid.length > 0 ) {
-      _showError();
+      event.preventDefault();
+      var msg = '';
+
+      validatedFields.invalid.forEach( function( validation ) {
+        msg += validation.label + validation.msg + '</br>';
+      } );
+      _setNotification( _notification.ERROR, msg ||
+                        ERROR_MESSAGES.DEFAULT );
+    }
+  }
+
+  /**
+   * Validate the fields of our form.
+   * @param {Field} field A form field.
+   * @param {String} selector Selector used to retreive the dom element.
+   * @param {Boolean} isInGroup Flag used determine if field is in group.
+   * @returns {String} The label of the field.
+   */
+  function _getLabelText( field, selector, isInGroup ) {
+    var labelText = '';
+    var labelDom;
+
+    if ( isInGroup && !selector ) {
+      labelDom = getClosestElement( field, 'fieldset' );
+      if ( labelDom ) labelDom = labelDom.querySelector( 'legend' );
     } else {
-      _showSuccess();
+      selector = selector ||
+                 'label[for="' + field.getAttribute( 'id' ) + '"]';
+      labelDom = _form.querySelector( selector );
     }
+
+    if ( labelDom ) labelText = labelDom.textContent.trim();
+
+    return labelText;
   }
 
   /**
-   * Show error notification.
-  */
-  function _showError() {
-    _notification.setTypeAndContent( _notification.ERROR, 'Error!' );
-    _notification.show();
+   * Set the notification type, msg, and visibility.
+   * @param {String} type The type of notification to display.
+   * @param {String} msg The message to display in the notification.
+   * @param {String} methodName The method to use to control visibility
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/f217937d4eec3537f28765b2f956d29b1609df50